files/image_config/sysmonitor/sysmonitor.py

```diff
@@ -0,0 +1,691 @@
+#!/usr/bin/python3
+
+from datetime import datetime
+import time
+import datetime
+import os
+import subprocess
+import sys
+import threading
+import syslog
+import argparse
+import multiprocessing as mp
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicV2Connector
+import socket
+import json
+import fcntl
+import stat
+
+SYSLOG_IDENTIFIER="system#monitor"
+STATE_FEATURE_TABLE_NAME = "FEATURE"
+REDIS_TIMEOUT_MS = 0
+SYSTEM_STATE="DOWN"
+logger = None
+SYSTEM_CORESRV_STATE="DOWN"
+SYSTEM_ALLSRV_STATE="DOWN"
+SYSREADY_LOCKFILE="/var/run/sysready.lock"
+core_dnsrvs_name_list=[]
+dnsrvs_name_list=[]
+allsrvs_dict={}
+coresrvs_dict={}
+allsrvs_status="DOWN"
+coresrvs_status="DOWN"
+spl_srv_list= ['database-chassis', 'gbsyncd']
+core_srv_list = [
+        'swss.service',
+        'bgp.service',
```

Comment:
i am not sure bgp.service is a core service, we need to check how this interack with application extension framework.

Reply:
We will probably need to define whether a docker is a core or not in the docker's manifest.